### ISSUE TITLE
Ooi_2007: Processing submission

### ISSUE THREAD

**Issue by CaitlanB** [traitecoevo/austraits.build#212]
**Source**: unpublished
**Custodian**: Mark Ooi
**Handled by**: Caitlan

- [x] Contact made with contributor (label as `waiting on response` if waiting for files)
- [x] Files received from contributor saved in google drive folder `austraits/data submissions/new`  ( label as `ready` )
- [x] Started importing files into own Austraits branch  (mark label as `In progress` and assign issue to relevant person )
- [x] Metadata complete, builds into Austraits
- [x] Report reviewed by AusTraits team
- [x] Report sent to contributor ( label as `waiting on response` )
- [x] Response received from contributor
- [x] Files updated & report finalised
- [x] Final checks ( all tests pass )
- [x] Branch merged into master branch (issue `closed` )
